### PR TITLE
reorg: move JS ops implementations to cli/js/ops/, part 2

### PR DESCRIPTION
**Pull request by bartlomieju**

### CODE REVIEW

**Comment by bartlomieju** [comment]
After this PR there are 3 more files to create in `cli/js/ops`:
- `fs.ts`
- `net.ts`
- `tls.ts`

File: cli/js/dispatch_minimal_test.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ const readErrorStackPattern = new RegExp(
   `^.*
     at unwrapResponse \\(.*dispatch_minimal\\.ts:.*\\)
     at Object.sendAsyncMinimal \\(.*dispatch_minimal\\.ts:.*\\)
-    at async Object\\.read \\(.*files\\.ts:.*\\).*$`,
+    at async Object\\.read \\(.*io\\.ts:.*\\).*$`,
   "ms"
 );
```
Comment (ry) [inline cli/js/dispatch_minimal_test.ts]: Not actionable for this PR, just a comment: This sort of call stack inspection, especially with an RegExp does not belong here. AFAIK async stack trace is broken anyway - so this test probably does nothing.

Comment (bartlomieju) [reply cli/js/dispatch_minimal_test.ts]: I'm not sure, this stack trace seems valid to me, I have a feeling that async stack traces are only broken in some conditions. Anyway, this needs to be sorted out 👍

**Comment by bartlomieju** [comment]
Ref #4248

**Pull request merged**